modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/odata/retriever/test/UserODataRetrieverCustomFieldsTest.java

```diff
@@ -241,6 +241,76 @@ public void testGetUsersFilterByCustomFieldWithEqualsAndDateTextType()
 		Assert.assertEquals(user1, users.get(0));
 	}
 
+	@Test
+	public void testGetUsersFilterByCustomFieldWithEqualsAndIntegerKeywordType()
```

Comment:
I'd add a test for `ExpandoColumnConstants.SHORT` and another for `ExpandoColumnConstants.LONG`, does it make sense for you?